.codesandbox/start.mjs

```diff
@@ -37,16 +37,19 @@ export function startSandbox(destination, buildAndWatch, installDeps = false) {
   const pathToTrigger = path.resolve(destination, 'package.json');
   subscribe((locked) => {
     if (!locked) {
-      const packageJSON = fs.readJsonSync(pathToTrigger);
-      fs.writeJSONSync(
+      const packageJSON = JSON.parse(fs.readFileSync(pathToTrigger, 'utf8'));
+      fs.writeFileSync(
         pathToTrigger,
-        {
-          ...packageJSON,
-          trigger: moment().format('YYYY-MM-DD HH:mm:ss'),
-        },
-        { spaces: 2 },
+        JSON.stringify(
+          {
+            ...packageJSON,
+            trigger: moment().format('YYYY-MM-DD HH:mm:ss'),
```

Comment:
i think we can also get rid of moment in favor of Intl constructor
moment is a huge lib, not worth having it for a trivial calls like this